examples/HeatMap.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2018 WorldWind Contributors
```

Comment:
Could you apply the same header as in https://github.com/NASAWorldWind/WebWorldWind/blob/develop/src/WorldWind.js to all files?

Reply:
Done

Reply:
This particular file still has the old copyright header. Please update it.